addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -250,7 +246,8 @@ def _prepare_down_payment_product_values(self):
             'invoice_policy': 'order',
             'company_id': self.company_id.id,
             'property_account_income_id': self.deposit_account_id.id,
-            'taxes_id': [Command.set(self.deposit_taxes_id.ids)],
+            'taxes_id': False,
+            'active': False,  # hide down payment product from user
```

Comment:
why create it, and then archive it?

Reply:
It's archived to not show up when the user interacts with the UI, but it is still used as down payment product.